vis/graphs.js

```diff
@@ -0,0 +1,221 @@
+            proj_type_quarter[6][sub], proj_type_quarter[7][sub], proj_type_quarter[8][sub],
+            proj_type_quarter[9][sub],proj_type_quarter[10][sub], proj_type_quarter[11][sub]],
+        type: 'bar',
+        name: 'App',
+    };
+    sub++;
+    var q6 = {
+        x: type_quarter,
+        y: [proj_type_quarter[0][sub], proj_type_quarter[1][sub], proj_type_quarter[2][sub],
+            proj_type_quarter[3][sub], proj_type_quarter[4][sub], proj_type_quarter[5][sub],
+            proj_type_quarter[6][sub], proj_type_quarter[7][sub], proj_type_quarter[8][sub],
+            proj_type_quarter[9][sub], proj_type_quarter[10][sub], proj_type_quarter[11][sub]],
+        type: 'bar',
+        name: 'Game',
+    };
+    sub++;
+    var q7 = {
+        x: type_quarter,
+        y: [proj_type_quarter[0][sub], proj_type_quarter[1][sub], proj_type_quarter[2][sub],
+            proj_type_quarter[3][sub], proj_type_quarter[4][sub], proj_type_quarter[5][sub],
+            proj_type_quarter[6][sub], proj_type_quarter[7][sub], proj_type_quarter[8][sub],
+            proj_type_quarter[9][sub], proj_type_quarter[10][sub], proj_type_quarter[11][sub]],
+        type: 'bar',
+        name: 'IoT',
+    };
+    sub++;
+    var q8 = {
+        x: type_quarter,
+        y: [proj_type_quarter[0][sub], proj_type_quarter[1][sub], proj_type_quarter[2][sub],
+            proj_type_quarter[3][sub], proj_type_quarter[4][sub], proj_type_quarter[5][sub],
+            proj_type_quarter[6][sub], proj_type_quarter[7][sub], proj_type_quarter[8][sub],
+            proj_type_quarter[9][sub], proj_type_quarter[10][sub], proj_type_quarter[11][sub]],
+        type: 'bar',
+        name: 'Writing',
+    };
+    sub++;
+    var q9 = {
+        x: type_quarter,
+        y: [proj_type_quarter[0][sub], proj_type_quarter[1][sub], proj_type_quarter[2][sub],
+            proj_type_quarter[3][sub], proj_type_quarter[4][sub], proj_type_quarter[5][sub],
+            proj_type_quarter[6][sub], proj_type_quarter[7][sub], proj_type_quarter[8][sub],
+            proj_type_quarter[9][sub], proj_type_quarter[10][sub], proj_type_quarter[11][sub]],
+        type: 'bar',
+        name: 'AR',
+    };
+    sub++;
+    var q10 = {
+        x: type_quarter,
+        y: [proj_type_quarter[0][sub], proj_type_quarter[1][sub], proj_type_quarter[2][sub],
+            proj_type_quarter[3][sub], proj_type_quarter[4][sub], proj_type_quarter[5][sub],
+            proj_type_quarter[6][sub], proj_type_quarter[7][sub], proj_type_quarter[8][sub],
+            proj_type_quarter[9][sub], proj_type_quarter[10][sub], proj_type_quarter[11][sub]],
+        type: 'bar',
+        name: 'VR',
+    };
+
+    var data = [q1, q2, q3, q4, q5, q6, q7, q8, q9, q10];
+
+    var layout_proj_quarters = {
+        title: 'Number of Projects by Type per Quarter',
+        xaxis: {
+            title: 'Quarters',
+        },
+        yaxis: {
+            title: 'Number of Projects',
+        },
+        barmode: 'stack',
```

Comment:
i think tech and nontech are encompassing fields, which means ios, iot, web all are a part of tech, and writing, art are a part of nontech

Reply:
i dont think this has been resolved yet!

you are counting up how many times we say something is 'tech' or 'iot' but generally, we will tag a project as 'tech' AND 'iot'.. this means ur double counting projects which leads to an inaccurate data visualization!